front/src/components/UI/Tabs/EditTabsButton/EditTabsButton.tsx

```diff
@@ -0,0 +1,10 @@
+import { EditButtonStyled } from './EditTabsButton.styled';
+import './EditTabsButton.styled.ts';
+
+const EditTabsButton = (props: { label: string}) => {
```

Comment:
jest ok, tylko wynieś typ do interfejsu. Łatwije będzie w przyszłości to rozbudowywać